### PR TITLE
Procedural Prefabs: Python documentation cleanup

### PR DESCRIPTION
**Pull request by amzn-mike**

### CODE REVIEW

File: AutomatedTesting/Editor/Scripts/scene_helpers.py [hunk 1]
```diff
@@ -14,30 +14,44 @@
...
+
+    Parameters
+    ----------
+    name :
+        String to clean.
+        
+
+    Returns
+    -------
+    type
```
Comment (jackalbe) [inline AutomatedTesting/Editor/Scripts/scene_helpers.py:34]: is type supposed to be 'str' here?

File: Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py [hunk 2]
```diff
@@ -7,13 +7,13 @@
 import typing
 import json
 import azlmbr.scene as sceneApi
-from enum import Enum, IntEnum
+from enum import IntEnum
```
Comment (jackalbe) [inline Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py:10]: this is pretty cool. good to know that enum is a valid Python concept

**Review by AMZN-stankowi — APPROVED** [review]

File: Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py [hunk 3]
```diff
@@ -96,6 +96,23 @@ def get_node_content(self, node):
...
+
+class TangentSpaceSource(IntEnum):
+    SCENE = 0
+    MIKKT_GENERATION = 1
+
+
+class TangentSpaceMethod(IntEnum):
+    TSPACE = 0
+    TSPACE_BASIC = 1
+
```
Comment (jackalbe) [inline Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py:114]: should these be documented as well?

File: Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py [hunk 4]
```diff
@@ -109,40 +126,125 @@ class DecompositionMode(IntEnum):
...
+        ----------
+        mesh_group :
+            Mesh group to apply the selection to.
+        node_name :
+            Path of the node.
+            
+
+        Returns
+        -------
+
```
Comment (jackalbe) [inline Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py:198]: maybe remove the Returns section?

File: Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py [hunk 5]
```diff
@@ -109,40 +126,125 @@ class DecompositionMode(IntEnum):
...
```
Comment (jackalbe) [inline Gems/PythonAssetBuilder/Editor/Scripts/scene_api/scene_data.py:215]: maybe remove any blank Returns section?

**Review by jackalbe — APPROVED** [review]

**Pull request merged**